src/updateAttribute.js

```diff
@@ -56,7 +56,13 @@ function setAttribute(node, attrName, attrValue, oldAttrValue, isSvgAttribute) {
 
       // if the event is getting added first time add a listener
     } else if (!oldAttrValue && attrValue) {
-      node.addEventListener(eventName, patchedHandler);
+      let isCaptureEvent = eventName.endWith('Capture');
+      if(isCaptureEvent) {
+        let cleanedEventName = eventName.replace('Capture', '');
+        node.addEventListener(cleanedEventName, patchedHandler);
+      } else {
```

Comment:
No need to have if/else, we can just do.
```js
node.addEventListener(cleanedEventName, patchedHandler, isCaptureEvent);
```

If it is false it will assign it in bubble phase.

Reply:
Yes that indeed is cleaner :+1: 
Will update this